server/src/envs/index.ts

```diff
@@ -99,12 +41,24 @@ function ensureEnvVariable(name: string): string {
 }
 
 function getVariables(): EnvVariables {
-  const network = ensureEnvVariable('ETH_NETWORK');
-  // const ownerAddress = ensureEnvVariable('POAP_OWNER_ADDR')
-  const ownerPK = ensureEnvVariable('POAP_OWNER_PK');
+  let provider: Provider;
+  let envProvider = ensureEnvVariable('PROVIDER');
```

Comment:
By adding this, we need to make sure that this variable in on production (`PROVIDER`). If the variable is not added, the `ensureEnvVariable` would stop the server.
I'm not totally convinced of using such a strong method, but in this cases lets at least comment in the PR that this variable needs to be added